### PR TITLE
Style and spell checks for all virtualization docs done

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some (quite a few) suggestions for these files. This is not the entire set, I'll deal with the rest on Tuesday. Thanks!

File: xml/xen_xmtoxl.xml [hunk 1]
```diff
@@ -4,231 +4,227 @@
...
-   applications.
-  </para>
+    <para>
+      One of the major differences between <command>xend</command> and libxl is
+      that the former is stateful, while the latter is stateless. With
+      <command>xend</command>, all client applications such as
+      <command>xm</command> and &libvirt; see the same system state.
+      <command>xend</command> maintains the state for the entire &xen; host. In
+      libxl, client applications such as <command>xl</command> or &libvirt;
+      must maintain state. Thus domains created with <command>xl</command> or
```
Comment (dariavladykina) [inline xml/xen_xmtoxl.xml]:
```suggestion
      must maintain state. Thus domains created with <command>xl</command> are
```

**Review by dariavladykina — COMMENTED** [review]
The final batch of suggestions, please

**Review by tbazant — COMMENTED** [review]
Thanks @dariavladykina , really a good job!

**Pull request merged**